### DISCUSSION TITLE
Constant high

### DISCUSSION THREAD

**Discussion by uaedude** [tribalfs/GalaxyMaxHzPub#204] [Ideas]
I have a question you know during gaming the connection of the cell changes from 5G to 4G plus it does that because of overheating and processing of the device however I think with your recording to constant high it's a possibility that you will be able to find out a way to make the phone understand that it is doing surfing instead of gaming so that 5G sticks thanks

**Comment by tribalfs** [comment]
Sorry, I don't know how to do that.
